docker_images_to_adg.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+#
+
+set -e # exit on failure
+
+# Pull the listed imaged
+while read -r line; do
+  echo "Pulling image ${line}"
+  docker pull "${line}" 
+done < "$1"
+
+
+target_date_name="$(date +%Y_%m_%d_%H_%M_%S)"
+
+target_dir="adg_${target_date_name}"
+
+docker_blob="docker_blob_${target_date_name}.tar"
+
+blob_dir="blob_dir_${target_date_name}"
+
+mkdir "${blob_dir}"
+
+echo "Saving images to ${blob_dir}/${docker_blob}"
+docker save "$(cat "$1")" -o "${blob_dir}/${docker_blob}"
```

Comment:
I quoted this in response to shellcheck, but the file with name in "$1" is a list of images, correct? So this should expand to `docker save image1 image2 image3 -o blob_dir/docker_blob` ?